image.yaml

```diff
@@ -151,8 +154,10 @@ artifacts:
       md5: 7c743e35463db5f55f415dd666d705c5
     - path: oauth-20100527.jar
       md5: 91c7c70579f95b7ddee95b2143a49b41
-    - path: activemq-rar-5.11.0.redhat-621084.rar
-      md5: 207e17ac8102c93233fe2764d1fe8499
+    - url: https://maven.repository.redhat.com/ga/org/apache/activemq/activemq-rar/5.11.0.redhat-630329/activemq-rar-5.11.0.redhat-630329.rar
+      md5: 76ed338d86bb3dbba2d4bd48daff7dbb
+    - path: hawkular-javaagent-1.0.1.Final-redhat-2-shaded.jar
```

Comment:
The hawkular jar should not be provided through the hawkular module, so you should be able to remove this entry.